pymmcore_plus/core/_mmcore_plus.py

```diff
@@ -724,6 +725,115 @@ def setShutterOpen(self, *args):
             state = args
         self.events.propertyChanged.emit(shutterLabel, "State", state)
 
+    def deleteConfig(self, group: str, preset: str) -> None:
+        super().deleteConfig(group, preset)
+        self.events.presetDeleted.emit(group, preset)

```

Comment:
even if the wording of micro-manager's API is sometimes confusing, I feel like we should stick strictly to it.

So, if someone is wondering what event might get emitted when the `deleteConfig` method is used, I think their first guess would be `configDeleted`

and `deleteConfigGroup` -> `configGroupDeleted`
etc...